builder-support/debian/authoritative/debian-buster/config/geoip.conf

```diff
@@ -1,8 +1,7 @@
 # geoip Configuration
 #
-# See for more information https://doc.powerdns.com/md/authoritative/backend-geoip/
+# See for more information https://docs.powerdns.com/authoritative/backends/geoip.html
```

Comment:
nit: all other files in git use doc.powerdns.com, not doc**s**

Reply:
Fair enough, I just pushed a commit to use doc.powerdns.com instead.